src/Gui/DlgExpressionInput.cpp

```diff
@@ -156,7 +156,7 @@ void DlgExpressionInput::textChanged(const QString &text)
             //set default palette as we may have read text right now
             ui->msg->setPalette(ui->okBtn->palette());
 
-            NumberExpression * n = Base::freecad_dynamic_cast<NumberExpression>(result.get());
+            auto n = Base::freecad_dynamic_cast<NumberExpression>(result.get());
```

Comment:
This is actually a case where I'd be inclined to keep the `*`, as a reminder that you aren't just getting a `NumberExpression`, but a pointer to one.

Reply:
ok